cli/node.go

```diff
@@ -143,32 +141,21 @@ var startNodeCmd = &cobra.Command{
 			},
 			UDPPort:     udpPort,
 			TCPPort:     tcpPort,
-			TopicName:   validatorKey,
 			HostDNS:     hostDNS,
 			HostAddress: hostAddress,
 		}
-		network, err := p2p.New(cmd.Context(), logger, &cfg)
+		network, err := p2p.New(cmd.Context(), logger, &validatorStorage, &cfg)
 		if err != nil {
 			logger.Fatal("failed to create network", zap.Error(err))
 		}
 
-		msgQ := msgqueue.New()
-
 		ssvNode := node.New(node.Options{
 			ValidatorStorage: validatorStorage,
+			IbftStorage:      ibftStorage,
```

Comment:
You only use it to setup validators, no need to pass pointer to db or have it in the struct.. it makes the code more spaghetti like.